CHANGELOG.md

```diff
@@ -162,9 +165,6 @@ v0.36.2 (2023-09-22)
 
 - Fix `loki.source.file` race condition in cleaning up metrics when stopping to tail files. (@thampiotr)
 
-- Fixed the `agent_prometheus_scrape_targets_gauge` incorrectly reporting all discovered targets
-  instead of targets that belong to current instance when clustering is enabled. (@thampiotr)
-
```

Comment:
I think it's best to retract this statement from `0.36.2` - the bugfix was incorrect, so the metric is still not correct.